src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpAndCollectorStep.java

```diff
@@ -22,7 +24,7 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
         List<String> listOfSaeFileNames = getConcurStandardAccountingExtractService().buildListOfFileNamesToBeProcessed();
         boolean success = true;
         for (String saeFileName : listOfSaeFileNames) {
-            LOG.info("execute, processing: " + saeFileName);
+            LOG.debug("execute, processing: " + saeFileName);
```

Comment:
I do log it with info elsewhere, so this turns out to be a duplicate log message in most cases.  I wanted to leave it as debug in case we need to trace things more indepthly.